### PR TITLE
Optionally skip the creation of parent files list

### PR DESCRIPTION
**Pull request by mpuccio**
This will prevent the creation of huge parent files maps in the case of
slimmed derived dataset creation

### CODE REVIEW

**Comment by mpuccio** [comment]
@jgrosseo, as discussed over MM this should fix the merging issues we got for the latest (hyper)nuclei hyperloop trains on the large dataset.

File: Common/Core/aodMerger.cxx [hunk 1]
```diff
@@ -61,13 +63,16 @@ int main(int argc, char* argv[])
...
+      skipParentFilesLists = true;
     } else if (c == 5) {
+      verbosity = atoi(optarg);
+    } else if (c == 6) {
       printf("AO2D merging tool. Options: \n");
       printf("  --input <inputfile.txt>      Contains path to files to be merged. Default: %s\n", inputCollection.c_str());
       printf("  --output <outputfile.root>   Target output ROOT file. Default: %s\n", outputFileName.c_str());
       printf("  --max-size <size in Bytes>   Target directory size. Default: %ld. Set to 0 if file is not self-contained.\n", maxDirSize);
       printf("  --skip-non-existing-files    Flag to allow skipping of non-existing files in the input list.\n");
+      printf("  --skip-parent-files-lists    Flag to allow skipping the creation of the parent files list.\n");
```
Comment (jgrosseo) [inline Common/Core/aodMerger.cxx:75]:
```suggestion
      printf("  --skip-parent-files-list     Flag to allow skipping the merging of the parent files list.\n");
```

File: Common/Core/aodMerger.cxx [hunk 2]
```diff
@@ -44,8 +45,9 @@ int main(int argc, char* argv[])
     {"output", required_argument, nullptr, 1},
     {"max-size", required_argument, nullptr, 2},
     {"skip-non-existing-files", no_argument, nullptr, 3},
-    {"verbosity", required_argument, nullptr, 4},
-    {"help", no_argument, nullptr, 5},
+    {"skip-parent-files-lists", no_argument, nullptr, 4},
```
Comment (jgrosseo) [inline Common/Core/aodMerger.cxx:48]:
```suggestion
    {"skip-parent-files-list", no_argument, nullptr, 4},
```

**Review by jgrosseo — COMMENTED** [review]
Thanks! Please consider the two small changes. We can merge it later today and try it tomorrow (needs adjustment in Hyperloop of course...)

**Comment by mpuccio** [comment]
@jgrosseo, thanks! Comments implemented!

**Review by jgrosseo — APPROVED** [review]

**Pull request merged**